spring-boot-autoconfigure/src/main/java/org/axonframework/boot/AxonAutoConfiguration.java

```diff
@@ -70,11 +70,8 @@
 @EnableConfigurationProperties(EventProcessorProperties.class)
 public class AxonAutoConfiguration {
 
-    private final EventProcessorProperties eventProcessorProperties;
-
-    public AxonAutoConfiguration(EventProcessorProperties eventProcessorProperties) {
-        this.eventProcessorProperties = eventProcessorProperties;
-    }
+    @Autowired
```

Comment:
Spring recommends constructor injection instead of field injection. Is there a specific reason for this change?